### ISSUE TITLE
Markdown editor incorrectly sets initial height

### ISSUE THREAD

**Issue by jwohlfert23** [laravel-filament/filament#1633]
### Package

filament/filament

### Package Version

v2.10.10

### Laravel Version

v9.1.0

### Livewire Version

v2.10.3

### PHP Version

PHP 8.0.14

### Bug description

<img width="1328" alt="Screen Shot 2022-02-16 at 9 38 22 AM" src="https://user-images.githubusercontent.com/2797531/154287440-8c6953bd-9542-4d68-95d0-64dd0693717a.png">

In this example, the original height of the "overlay" block is incorrectly set to 148px.  This results in all of that HTML overflowing into the below elements of the page.   When a change is made to the component's' state (deleting a character in the text box for example), the height is then correctly calculated to 1000px+ in this case.  Seems to me that when initial render call is made in `markdown-editor.js`, the contents of the textarea isn't there yet? 

Relavent code: 

```js
   init: function () {
                if (this.state !== null) {
                    this.render()
                }

                this.$watch('state', () => {
                    this.render()
                })
            },

            render: function () {
                if (this.$refs.textarea.scrollHeight > 0) {
                    this.$refs.overlay.style.height = '150px'
                    this.$refs.overlay.style.height = this.$refs.textarea.scrollHeight + 'px'
                }
```

### Steps to reproduce

Create a markdown field with an initial state that pushes the textarea over the min-height of 150px.

**Comment by mikerockett** [comment]
Confirmed, just noticed this after upgrading from 1.x:

<img width="1285" alt="image" src="https://user-images.githubusercontent.com/4586280/154842426-db5b640b-c9f9-4784-826b-a03dc862e1f4.png">

**Comment by danharrin** [comment]
Closed by #1699.

**Comment by voidgraphics** [comment]
Was this included in `v2.10.14`? To my observations based on the time of merging and the time of release, yes, but I am still experiencing the issue: text is overlapping strangely and also sticking out the bottom of the field. This is a brand new filament install with no custom css or anything of the sort.

<img width="857" alt="CleanShot 2022-02-27 at 20 55 42@2x" src="https://user-images.githubusercontent.com/9298484/155897665-ecea13b2-9168-49be-b106-b32e5ba0122d.png">

**Comment by daronspence** [comment]
I'm still seeing behavior similar to this on mobile devices. Will try to confirm later when I have a more stable connection.
